docs/managed-datahub/subscription-and-notification.md

```diff
@@ -156,6 +169,23 @@ You can unsubscribe from any asset to stop receiving notifications about it. On
 What if I want to be notified about different changes?
 </summary>
 To modify your subscription, use the dropdown menu next to the Subscribe button to modify the changes you want to be notified about.
+</details>
+<details>
+<summary>
+I want to configure multiple channels. How many slack channels or emails can I configure to get notified? 
+</summary>
+At the platform-level, you can configure one email and one slack channel.
+
+At the user and group -levels, you can configure one default email and slack channel as well as overwrite that email/channel when you
+go to a specific asset to subscribe to. 
+
+To configure multiple channels, as a prereq, ensure you have the appropriate privileges. And then:
+1. Create a datahub group for each channel you want notifications for. 
+2. Add yourself as a member to each of the groups.
+3. Go to each groups page (eg. through Settings > Users & Groups > Group Name OR by searching for your group's name).
+4. Click on 'Notifications', enable the type of notifications you want, and add the appropriate email or slack channel id.
```

Comment:
if you want for more simplicity: these two steps can be skipped now because when they subscribe to an asset we automatically turn on notifications and set default channel/email